website/remote-content/edge-proxy.js

```diff
@@ -29,11 +29,7 @@ const DOCS = mapObject(enrich)({
 
 const getAdmonitions = (data) => {
     const admonitions = {
-        'unleash-proxy': `:::tip
-
-Looking for how to run the Unleash proxy? Check out the [_how to run the Unleash proxy_ guide](../how-to/how-to-run-the-unleash-proxy.mdx)!
-
-:::`,
+        'unleash-proxy': ``,
```

Comment:
What happens if we remove the key? Rather than set an empty value

Reply:
it's the same thing, I left the keys in there so it would be easier to edit later if we need to inject some admonitions, but I can remove them